client/src/infographic/Infographic.yaml

```diff
@@ -61,6 +61,26 @@ infograph_explorer_link:
   en: Advanced search
   fr: Recherche avancée
 
+invalid_level_redirect_home:
+  transform: [handlebars]
+  en: The "{{level}}" is invalid. We redirected you to homepage.
+  fr: TODO
+invalid_subject_redirect_parent_dept:
+  transform: [handlebars]
+  en: The organization "{{subject_id}}" is invalid. We redirected you to its parent department.
```

Comment:
Your browser was attempting to visit one of our infographics. Unfortunately, we could not recognize "{{subject_id}}" as a valid infographic parameter. While "{{subject_id}}" itself couldn't be found, we were able to match the "{{dept_code}}" portion to a valid organization. We have redirected you to the infographic for that organization.

You were likely following a link intended to take you to an infographic for a core responsibility or program belonging to this organization. You can view and navigate to all of this organization's valid core responsibilities and programs under "Where can I go from here?" below.

If you believe something is incorrect on our end, we invite you to alert us using the Report a Problem form provided at the bottom of the page. Thank you!